sw/snRuntime/CMakeLists.txt

```diff
@@ -57,11 +57,11 @@ include_directories(
 # Common sources
 set(sources
     src/barrier.c
-    src/dma.c
+#    src/dma.c
```

Comment:
This should be handled without affecting  `snRuntime`. Not sure why we need to avoid these files now that we use GCC for the RISC-V tests, but either way, we should not touch the main runtime sources.